Tribler/Core/Modules/restapi/market/asks_bids_endpoint.py

```diff
@@ -1,5 +1,7 @@
 import json
```

Comment:
Please `import json_util as json` here so we can catch the nasty `UnicodeDecodeError`s